api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ElasticsearchService.scala

```diff
@@ -48,7 +50,7 @@ class ElasticsearchService(elasticClient: ElasticClient)(
       .request(
         queryOptions,
         index,
-        scored = queryOptions.searchQuery.isDefined
+        scored = queryOptions.searchQuery.isDefined || queryOptions.scoredFilters.nonEmpty
```

Comment:
It doesn't feel like it makes that much sense to have this on a filter per filter bases, and might lead to confusion.

Why not have it as an explicit value. on the `ElasticsearchQueryOptions`?

Reply:
The reason it feels odd is it is on a per request basis, and it feels hidden within the filters.

Reply:
I'm not sure I understand the comment about it not making sense on "filter per filter bases" but I agree that having it all live in `filters` / `DocumentFilter` feels odd (it was just the minimal change given how much the query config gets passed around through various case classes etc). That said, this is still a filter because it's a `must`, not a `should`, and so it's hard to know how to name it in such a way that indicates how it should be used...

Are you suggesting that we have `scoredFilters` and `unscoredFilters` as explicit fields that are populated "manually" on `ElasticsearchQueryOptions` and `ImagesSearchOptions`? Or something else?

Reply:
I am concerned as we have created abstractions on top of elastic DSL, which nearly always feel as though they can be expressed as part of the DSL.

Currently we have starting at the request =>
`QueryParams` => `SearchOptions`
`SearchOptions` => `ElasticsearchQueryOptions`
`ElasticsearchQueryOptions` => `ElasticsearchDSL`

I wonder if we had to start with what we want in Elastic i.e. and work up to the `SearchOptions` we might have something similar, if anything we could cut out `SearchOptions` => `ElasticsearchQueryOptions`.

For this instant we might have:
```
scala case SearchOptions(filters: List[Filter], musts:List[Musts], size: Int, sort: List[Sort])
```

Which can then be converted directly into ES queries.

In short, I would do what you've suggested explicitly saying these are `mustQueries` to avoid having to figure out what that infers further down the line.